### ISSUE TITLE
Update instructions to include `api = "0.2"` to buildpack.toml?

### ISSUE THREAD

**Issue by vazexqi** [heroku/cnb-shim#4]
While following the instructions on the README.md, the buildpack shim I created failed to run with the `pack` CLI because of this error:

```
 (Buildpack API version 0.1) is incompatible with lifecycle 0.6.1 (Buildpack API version 0.2)
```

I updated the buildpack.toml following https://buildpacks.io/docs/reference/buildpack-api/#example-2 to add `api = "0.2"` and that seems to work.

_Is this the right way?_ 

If so, should the instructions be updated?
If not, what would be the better way to address this issue?

**Comment by mraible** [comment]
I talked with @jkutner today and I believe you're correct. Also, I'm not sure the leading > are needed. On my Mac, I had to do:

```shell
cat > buildpack.toml << TOML
api = "0.2"

[buildpack]
id = "heroku.static"
version = "0.1"
name = "Static"

[[stacks]]
id = "heroku-18"
TOML
```

**Comment by jkutner** [comment]
Fixed by 66c3085244de920a861508a231bc399c97f85799
